index.js

```diff
@@ -51,87 +50,86 @@ function RFID (hardware, options, callback) {
   self.hardware = hardware;
   self.ready = false;
   self.irq = hardware.digital[2];
-  self.irqcallback = function () {
-    self.emit('irq', null, 0);
-  };
-  self.irq.on('fall', self.irqcallback);
+  self.irq.input();
+
   self.nRST = hardware.digital[1];
 
-  self.nRST.output();
-  self.nRST.low(); // Toggle reset every time we initialize
+  // Toggle reset every time we initialize
+  self.nRST.write(false, function continueInit() {
```

Comment:
I added a callback here to ensure we had written the RESET line low. This lets us be confident that the device has actually reset before we continue.